langgraph/graph/message.py

```diff
@@ -10,6 +11,13 @@
 
 from langgraph.graph.state import StateGraph
 
+
+class MessageModifier(TypedDict):
```

Comment:
@hinthornw is this similar to what you had in mind? as an alternative to `DeleteMessage`

Reply:
Ya!